src/cco-modules/InformationEntityOntology.ttl

```diff
@@ -26,6 +26,7 @@
 
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/condition_described_by
 cco:condition_described_by rdf:type owl:ObjectProperty ;
+                           rdfs:subPropertyOf cco:is_subject_of ;
```

Comment:
@mark-jensen  Should this be a subproperty of cco:described_by ?

Reply:
The relation is between a Performance Specification (a Directive ICE) and an entity. Making it a sub of `describes` would be inconsistent.

Reply:
I agree it's a tad odd given the label. This trouble re how ICEs can have parts that are different types than the whole, eg- an ICE that describes an event and has parts which name a person and location, will hopefully be addressed as part of the work improving CCOs model for information.